client/src/utils/index.ts

```diff
@@ -1,6 +1,9 @@
-// These regex are not for validation, it is purely to see
-// if we are looking at something like what we want to validate
-// before we try to validate
+/*
+ * This regex is not for validation, it is purely to see
+ * if we are looking at something like what we want to validate
+ * before we try to validate.
+ */
 export const maybeEmailRE = /.*@.*\.\w\w/;
 export const maybeUrlRE = /https?:\/\/.*\..*/;
 export const hasProtocolRE = /^http/;
+
```

Comment:
Just need to remove this extra blank line 🙂

Reply:
Just did, I squashed the commits and force pushed to my fork, waiting for tests to work

Reply:
@nhcarrigan Sorry for my "newbieness", I'm very new to linters, so I made some mistakes

Reply:
Happy to help! Looks like you've resolved the lint issues. Nice work 😁